### PR TITLE
Update import metrics and add verification time

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

File: primitives/consensus/common/src/import_queue.rs [hunk 1]
```diff
@@ -181,11 +185,22 @@ pub enum BlockImportError {
 }
 
 /// Single block import function.
-pub fn import_single_block<B: BlockT, V: Verifier<B>, Transaction>(
+pub(crate) fn import_single_block_unmetered<B: BlockT, V: Verifier<B>, Transaction>(
```
Comment (mxinden) [inline primitives/consensus/common/src/import_queue.rs:188]:
```suggestion
#[cfg(test)]
pub(crate) fn import_single_block_unmetered<B: BlockT, V: Verifier<B>, Transaction>(
```

File: primitives/consensus/common/src/metrics.rs [hunk 2]
```diff
@@ -34,6 +39,42 @@ impl Metrics {
 				)?,
 				registry,
 			)?,
+			block_verification_time: register(
+				HistogramVec::new(
+					HistogramOpts::new(
+						"block_verification_time",
+						"Histogram of time taken to import block",
```
Comment (mxinden) [inline primitives/consensus/common/src/metrics.rs:46]:
```suggestion
						"Histogram of time taken to import blocks",
```

File: primitives/consensus/common/src/import_queue/basic_queue.rs [hunk 3]
```diff
@@ -146,11 +148,6 @@ struct BlockImportWorker<B: BlockT, Transaction> {
 	_phantom: PhantomData<Transaction>,
 }
 
-const METRIC_SUCCESS_FIELDS: [&'static str; 8] = [
```
Comment (mxinden) [inline primitives/consensus/common/src/import_queue/basic_queue.rs:149]: :pray:

File: primitives/consensus/common/src/metrics.rs [hunk 4]
```diff
@@ -34,6 +39,42 @@ impl Metrics {
...
+					),
```
Comment (mxinden) [inline primitives/consensus/common/src/metrics.rs:47]: By default the Prometheus client library will use the following buckets:

`0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1.0, 2.5, 5.0, 10.0,` [see crate](https://docs.rs/prometheus/0.7.0/src/prometheus/histogram.rs.html#33-35).

I am not familiar with the timings here. Is this a good fit for this histogram?

File: primitives/consensus/common/src/import_queue.rs (hunk 1 above)
Comment (NikVolf) [reply primitives/consensus/common/src/import_queue.rs:188]: Unfortunately it is used in another crate and cannot be `#[cfg(test)]`

File: primitives/consensus/common/src/metrics.rs (hunk 4 above)
Comment (NikVolf) [reply primitives/consensus/common/src/metrics.rs:47]: Yeah, whatever is under 5ms is not very important 

I think minimum block imported is around 10ms

**Pull request merged**